src/azure-cli/azure/cli/command_modules/netappfiles/custom.py

```diff
@@ -522,3 +532,343 @@ def patch_subvolume(client, resource_group_name, account_name, pool_name, volume
         size=size
     )
     return client.begin_update(resource_group_name, account_name, pool_name, volume_name, subvolume_name, body)
+
+
+# ---- VOLUME GROUPS ----
+def create_volume_group(cmd, client, resource_group_name, account_name, pool_name, volume_group_name, vnet, ppg,
+                        sap_sid, subnet='default', location=None, tags=None, gp_rules=None, memory=100,
+                        add_snapshot_capacity=50, start_host_id=1, number_of_hots=1, prefix="", system_role="PRIMARY",
+                        data_size=None, data_throughput=None, log_size=None, log_throughput=None, shared_size=None,
+                        shared_throughput=None, data_backup_size=None, data_backup_throughput=None,
+                        log_backup_size=None, log_backup_throughput=None, backup_nfsv3=False, no_wait=False,
+                        data_repl_skd=None, data_src_id=None, shared_repl_skd=None, shared_src_id=None,
+                        data_backup_repl_skd=None, data_backup_src_id=None, log_backup_repl_skd=None,
+                        log_backup_src_id=None):
+    if number_of_hots < 1 or number_of_hots > 3:
+        raise CLIError("Number of hosts must be between 1 and 3")
```

Comment:
CLIError are not used any more. Please use other error types in this [guideline](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md)